docs/theory/statistics.rst

```diff
@@ -132,6 +134,21 @@ specify independent log-normal priors for the following quantities:
 - enzyme concentrations
 - unbalanced metabolite concentrations
 
+The distinction between balanced and unbalanced metabolites is also found in
+the statistical model. Where information about the unbalanced metabolites
+can be parsed in the form of a prior, whereas the balanced metabolites are always evaluated
+as a model likelihood. This distinction becomes aparent when considering what
+the unbalanced metabolites represent, which is a boundary condition. These
+define the outcome of systems of differential equations, in the case of Maud
+this happens to be balanced metabolite concentrations and fluxes. And, our
+knowledge about the state of each condition is only conveyed through priors
```

Comment:
Substantive comment: I think the distinction is a little less principled than it sounds from this passage. The bottom line is that we don't express priors on balanced metabolite concentrations because it is technically difficult to do so without introducing a bias, thanks to the transformed parameters issue. If we could - say if Stan introduced some way of automatically adding a Jacobian adjustment - then there would be no reason not to add priors on balanced metabolites too.